Source/Core/VideoBackends/OGL/OGLConfig.cpp

```diff
@@ -351,6 +347,21 @@ bool PopulateConfig(GLContext* m_main_gl_context)
   g_Config.backend_info.bSupportsTextureQueryLevels =
       GLExtensions::Supports("GL_ARB_texture_query_levels") || GLExtensions::Version() >= 430;
 
+  if (GLExtensions::Supports("GL_ARB_shader_storage_buffer_object"))
+  {
+    GLint vs = 0;
+    GLint fs = 0;
+    glGetIntegerv(GL_MAX_VERTEX_SHADER_STORAGE_BLOCKS, &vs);
+    glGetIntegerv(GL_MAX_FRAGMENT_SHADER_STORAGE_BLOCKS, &fs);
+    g_Config.backend_info.bSupportsFragmentStoresAndAtomics = fs >= 1;
+    g_Config.backend_info.bSupportsVSLinePointExpand = vs >= 1;
```

Comment:
The most nitpicky of nitpicks, but I think this would be better with consistently putting `fs` first and then `vs`; currently it's `vs` `fs` `vs` `fs` `fs` `vs` which is a bit confusing.

```suggestion
    GLint fs = 0;
    GLint vs = 0;
    glGetIntegerv(GL_MAX_FRAGMENT_SHADER_STORAGE_BLOCKS, &fs);
    glGetIntegerv(GL_MAX_VERTEX_SHADER_STORAGE_BLOCKS, &vs);
    g_Config.backend_info.bSupportsFragmentStoresAndAtomics = fs >= 1;
    g_Config.backend_info.bSupportsVSLinePointExpand = vs >= 1;
```

Reply:
Makes sense
I wish GitHub's "commit suggestion" button had an option to squash it into an existing commit...